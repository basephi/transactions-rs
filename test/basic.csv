type,client,tx,amount
deposit,1,1,1.0
deposit,2,2,2.0
deposit,4,4,6
deposit,3,3,9
withdrawal,1,5,0.5
withdrawal,2,6,0.005
dispute,4,4,0
dispute,3,3,0
chargeback,3,3,0
resolve,4,4,0
