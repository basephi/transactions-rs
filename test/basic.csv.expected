client,available,held,total,locked
1,0.5,0.0000,0.5,false
2,1.995,0.0000,1.995,false
3,0,0,0,true
4,6,0,6,false
